charts/drone-runner-docker/templates/deployment.yaml

```diff
@@ -152,3 +152,4 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      terminationGracePeriodSeconds: {{ .Values.terminationGracePeriodSeconds }}
```

Comment:
hey @Funghorn - Can we have a default value set to 30 seconds if the value is not provided?

Reply:
There is a default set already — https://github.com/drone/charts/blob/master/charts/drone-runner-docker/values.yaml#L109. If we want it changed from 3600 to 30, let me know.

Reply:
@jimsheldon @rajatharanganath any news? Should I change the existing default value or anything else?